common/api/core-geometry.api.md

```diff
@@ -6095,12 +6096,14 @@ export type XAndY = Readonly<WritableXAndY>;
 // @public
 export class XY implements XAndY {
     protected constructor(x?: number, y?: number);
+    at(index: number): number;
     static crossProductToPoints(origin: XAndY, targetA: XAndY, targetB: XAndY): number;
     distance(other: XAndY): number;
     distanceSquared(other: XAndY): number;
     freeze(): Readonly<this>;
+    indexOfMaxAbs(): number;
     isAlmostEqual(other: XAndY, tol?: number): boolean;
-    isAlmostEqualMetric(other: XAndY): boolean;
+    isAlmostEqualMetric(other: XAndY, radianTol?: number): boolean;
```

Comment:
Should be `distanceTol`, as in a tolerance in meters.

Reply:
will fix